services/121-service/src/events/events.controller.ts

```diff
@@ -91,19 +91,20 @@ export class EventsController {
         throw new HttpException('Forbidden', HttpStatus.FORBIDDEN);
       }
 
-      const result = await this.eventService.getEventsXlsxDto(
+      const result = await this.eventService.getEventsAsXlsx({
         programId,
         searchOptions,
-      );
+      });
       if (result.length === 0) {
         throw new HttpException({ errors: errorNoData }, HttpStatus.NOT_FOUND);
       }
       return sendXlsxReponse(result, filename, res);
     }
-    const result = await this.eventService.getEventsJsonDto(
+    // ##TODO: Why only check for format xlsx if the user has access to this program and not for json as well? If json is not used at the moment: easiest to just remove it here?
```

Comment:
It's used in the API tests (getEvents in registration helper). Mainly because it's more convienent to test with JSON than csv

Reply:
Permission check can be removed as it is already done with the decorator   @AuthenticatedUser({
    permissions: [PermissionEnum.RegistrationPersonalEXPORT],
  })